src/openms/include/OpenMS/ANALYSIS/ID/SiriusMSConverter.h

```diff
@@ -50,32 +50,34 @@ namespace OpenMS
   // struct to store information about accessions
   struct AccessionInfo
   {
-    String sf_path;
-    String sf_type;
-    String sf_accession;
-    String native_id_accession;
-    String native_id_type;
+    String sf_path; //< sourcefile path for mztab-m
+    String sf_type; //< sourcefile type for mztab-m
+    String sf_accession; //< sourcefile accessions for mztab-m
+    String native_id_accession; //< nativeID accession for mztab-m
+    String native_id_type; //< nativeID type for mztab-m
```

Comment:
that should be '///<`, see http://www.doxygen.nl/manual/docblocks.html#memberdoc